src/webextension/list/components/item-list.css

```diff
@@ -19,3 +19,21 @@
   border-bottom: 1px solid #d7d7db;
   background-color: #ffffff;
 }
+
+.item:hover {
+  background-color: #ededf0;
+}
```

Comment:
I'm going to have to think a bit more about the changes in this file. The style rules for the item lists are where they are now to minimize momentary flashes of color when opening the item details view in the doorhanger. These changes may regress that, but I need to sit down and go through all the behaviors carefully to be sure I'm getting this right before I tell you to do anything different. :)